docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart" %}
+	 <input type="submit" value="clear" {{id}}__clear" />
+ {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<form action="/sites/cart" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="_method" value="delete" autocomplete="off">
+    <input type="hidden" name="return_to" id="return_to" value="https://www.creator-website.com/book-tickets" autocomplete="off">
+    <input type="submit" value="Clear" class="button btn--outline cart-block__cart-clear">
+</form>
+```
+{% endraw %}
+
+##### redirect
+The Clear Cart Tag will redirect to the cart page by default.
+See the `return_to:` param to customise.
+
+##### cart ID
+The HTML input tag requires the id of the cart it is intended to clear.  
+
+##### Extra Params
+* `return_to:` To specify the redirect location, pass a URL as the value of the return_to: param.
+The following example redirects to the homepage. 
```

Comment:
Could we add examples of how the redirect, cart id, and return_to parameters would be used?

Reply:
Thanks for the notes @tomchipchase

I think we're good to go... see comments below.

Cart ID:  I thought we needed to specify the cart ID, but @paulina-a has clarified that's not the case, so I've removed the reference. 

Redirect: This happens by default. It's not a param that can be passed in or removed, it can only be overwritten with the return_to param.  added some info on how this works online 31.

Return_to: this is documented